scripts/build_lib.sh

```diff
@@ -19,22 +19,24 @@ toggle_failpoints() {
   mode="$1"
   if command -v gofail >/dev/null 2>&1; then
     run gofail "$mode" server/etcdserver/ server/storage/backend/
-    (
-      cd ./server
-      run go get go.etcd.io/gofail/runtime
-    ) || exit 2
-    (
-      cd ./etcdutl
-      run go get go.etcd.io/gofail/runtime
-    ) || exit 2
-    (
-      cd ./etcdctl
-      run go get go.etcd.io/gofail/runtime
-    ) || exit 2
-    (
-      cd ./tests
-      run go get go.etcd.io/gofail/runtime
-    ) || exit 2
+    if [[ "$mode" == "enable" ]]; then
```

Comment:
Please move `run gofail "$mode" server/etcdserver/ server/storage/backend/` (line 21) inside the `if` statement as well.

Reply:
`run gofail "$mode" server/etcdserver/ server/storage/backend/` supports both `mode=enable` and `mode=disable`. Then mode is disable gofail will revert the failpoint code back to comments. I think this is still useful.

Reply:
Makes sense.